packages/server-ct/src/server-ct.ts

```diff
@@ -58,8 +59,8 @@ export class Server {
     return app
   }
 
-  createRoutes (...args) {
-    return require('./routes-ct').apply(null, args)
+  createRoutes (...args: unknown[]) {
+    return initializeRoutes.apply(null, args)
```

Comment:
Did this mean that the require was actually delegated to later?
So the file was not required until createRoutes was called.

> EDIT: Since there is no global imports and this code is run on server, it should not be a bug. It might be a perf hit though. To keep a close eye on.